web/packages/teleterm/src/ui/initUi.ts

```diff
@@ -56,9 +56,15 @@ function notifyAboutStoredConfigErrors(
 ): void {
   const errors = configService.getStoredConfigErrors();
   if (errors) {
+    const isKeymapError = errors.some(e =>
+      e.path[0].toString().startsWith('keymap.')
```

Comment:
I'd extract the predicate function alone to `configService.ts` so that if we ever need to change it we'll be less likely to forget to update it. Here we could then just call `errors.some(isKeymapError)`

Reply:
Ah, it results in `Module not found: Can't resolve 'fs' in..`, because there is some nodejs stuff in `ConfigService`. We can create a new file `config/isKeymapError`, but I don't it is worth for so small function.

Reply:
Ah that's unfortunate. Yeah, I agree, maybe let's leave a comment in config service that the keymap. prefix is used in initUi in a predicate function.